tox.ini

```diff
@@ -85,6 +84,13 @@ changedir = doc
 commands =
     sphinx-build -q -E -b doctest . {envtmpdir}/doctest {posargs}
 
+[testenv:py3-pymongo-v3.12]
+commands =
+    pip install git+https://github.com/mongodb/mongo-python-driver.git#egg=pymongo[encryption]
```

Comment:
Doesn't this install pymongo@master? Should we use `git clone` or `pip install https://github.com/mongodb/mongo-python-driver/archive/v3.12.tar.gz` instead?